packages/shell-api-types/bin/compile-shell-api-types.js

```diff
@@ -190,9 +190,20 @@ const loadAll = () => {
   const main = fs.readFileSync(path.join(yamlDir, 'main.yaml'));
   const mainLib = yaml.load(main);
   const FILES = fs.readdirSync(yamlDir).filter((s) => (/[A-Z]/.test( s[0])));
-  let exports = '\nexport default ShellApi;\n';
-  const types = ['const unknown = {\n  type: \'unknown\',\n  attributes: {}\n}'];
-  const typeConsts = ['unknown'];
+  let exports = '';
+  const types = [
```

Comment:
(@mcasimir this is where I was commenting.)

This irritates me slightly - because the ShellAPI class is no longer generated from YAML, but it still needs to be present in the `shell-types-signatures` file, we have to define the attributes in two separate places (once in `shell-types-signatures` and once in `shell-api/src/shell-api.ts`. This is because the signatures file is generated also from YAML but ShellApi is no longer in YAML. Perhaps we can get rid of the need to have ShellAPI in the `shell-types-signatures` file entirely to fix the issue.

Reply:
Actually - nevermind, I just moved it out of this file so the issue should be fixed. It was only being used in the cli-repl completer module I believe.

The change is here: https://github.com/mongodb-js/mongosh/commit/d7c84b4675df0784083cce17465d7cecd687b215